controllers/topics-controllers.js

```diff
@@ -54,6 +55,25 @@ exports.getCommentsByArticleId = (req, res, next) => {
     });
 };
 
+exports.postComment = (req, res, next) => {
+    const { article_id } = req.params;
+    const newComment = {
+        ...req.body,
+        article_id,
+    }
+    if (newComment.body) {
+        insertComment(newComment).then((comment) => {
+            res.status(201).send({ comment });
+        }).catch((err) => {
+            next(err);
+        });
+    } else {
+        res.status(400).send({
+            'msg': 'Comment undefined'
+        });
+    }
```

Comment:
I would consider letting your error handling middleware cover the error response on line 71 rather than directly sending it in your controller. to let your middleware handle it you'd have to promise.reject in your model and let the controller catch it to pass it elsewhere. This would mean you'd need to move your logic (that checks if the comment body is defined) out the controller and into the model